python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -1230,6 +1230,20 @@ def my_apply(df, unused):
     tm.assert_series_equal(result, expected)
 
 
+@pytest.mark.parametrize("op", [operator.isub, operator.iadd])
+def test_isub_iadd(op):
+    xarray1 = xpd.array([10, 11, 12])
+    xarray2 = xpd.array([1, 2, 3])
+
+    array1 = pd.array([10, 11, 12])
+    array2 = pd.array([1, 2, 3])
+
+    xarray1 = op(xarray1, xarray2)
+    array1 = op(array1, array2)
```

Comment:
I'd like to call `getattr(xarray1, op)(xarray2)` with `op == "__iadd__"` or `"__isub__"` to avoid any `operator` module shenanigans. It surprises me that the tests passed with the typo `__idd__` above, so I want to guard against any other errors by calling the actual dunder-method on the object rather than dispatch through `operator`.

Reply:
Yup, done.